mempool/src/interface/mempool_interface.rs

```diff
@@ -67,6 +67,9 @@ pub trait MempoolInterface: Send + Sync {
 
     /// Set the maximum mempool size
     fn set_max_size(&mut self, max_size: MempoolMaxSize) -> Result<(), Error>;
+
+    /// Get current fee rate
```

Comment:
I think this is the upper-bound of the feerate, right?

Besides that, when returning a raw value, u128, it's always helpful to add the units. I believe it's atoms/kB, right?

Reply:
replaced with the lowest feerate to be included in the first X MB in the mempool, hardcoded to 5MB currently.

Reply:
Please add some more documentation